etc/tests/test_bootstrapper.py

```diff
@@ -187,17 +231,24 @@ def test_process_metrics_method_no_metadata_file(monkeypatch, s3_setup, tmpdir):
                      'outputs': 'test-file/test-file-copy.txt;test-file/test,file/test,file-copy.txt',
                      'user-volume-path': None}
 
-    output_path = Path('/tmp')
-    if is_writable_dir(output_path) is False:
-        pytest.skip('Not supported in this environment')
-
+    output_path = Path(tmpdir)
+    # metadata file name and location
     metadata_file = output_path / 'mlpipeline-ui-metadata.json'
+    # remove file if it already exists
+    remove_file(metadata_file)
 
-    os.remove(metadata_file)
+    try:
+        # override the default output directory to make this test platform
+        # independent
+        monkeypatch.setenv('ELYRA_WRITABLE_CONTAINER_DIR', str(tmpdir))
+        main_method_setup_execution(monkeypatch, s3_setup, tmpdir, argument_dict)
+    except Exception:
+        raise
+    finally:
+        monkeypatch.delenv('ELYRA_WRITABLE_CONTAINER_DIR')
```

Comment:
I don't think this finally block is necessary since each pytest test runs in its own env, temp dirs, etc.  Did you find otherwise?

Reply:
No, but I wanted to be explicit. I removed the code in all location.